tests/regression/lp-1813365/task.yaml

```diff
@@ -1,5 +1,5 @@
 summary: Regression check for https://bugs.launchpad.net/snapd/+bug/1813365
-systems: [ubuntu-*, ubuntu-core-*, debian-*]
+systems: [ubuntu-1*, ubuntu-2*, ubuntu-core-*, debian-*]
```

Comment:
This looks valid but is it related?

Reply:
If the system is ubuntu-* then ubuntu-secureboot-20.04-64 also matches and we don't want that, other tests are filtering like this (ubuntu-1*, ubuntu-2*) already and I copied from them.

Reply:
Ah, I msised that the original didn't have a digit prefix.